### PR TITLE
Create Buildah issue template

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Create a Buildah Issue template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+**Describe the results you received:**
+
+
+**Describe the results you expected:**
+
+
+**Output of `buildah version`:**
+
+```
```
Comment (rhatdan) [inline .github/ISSUE_TEMPLATE.md]: Also need the OS/Kernel Version/rpm -q buildah
cat /etc/containers/storage.conf

Comment (TomSweeneyRedHat) [reply .github/ISSUE_TEMPLATE.md]: Some of this was requested in the description section.  I've removed it from there and have added the above suggestions in the same manner as buildah version.  Please let me know if it needs further tweaking.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+3.
+
+
+**Describe the results you received:**
+
+
+**Describe the results you expected:**
+
+
+**Output of `rpm -q buildah`:**
```
Comment (rhatdan) [inline .github/ISSUE_TEMPLATE.md]: Might want to mention apt here also.  Otherwise LGTM

Comment (TomSweeneyRedHat) [reply .github/ISSUE_TEMPLATE.md]: Done.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by TomSweeneyRedHat** [comment]
@ripcurld0 I think we'll skip the "other useful info" field for now.  I think we're currently asking for as much info as I'm comfortable with and hopefully folks will put the other info in their description.  Let's see how the reports are filled out for a bit, then we can adjust as necessary.

**Comment by ripcurld0** [comment]
@TomSweeneyRedHat no problem.
I just was wondering if you considered it as an option.

Thanks for the feedback.

**Pull request closed without merging**